src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -1327,3 +1343,8 @@ def execute_flow(
             # remove line_number from output
             line_result.output.pop(LINE_NUMBER_KEY, None)
         return line_result
+
+
+def signal_handler(sig, frame):
```

Comment:
If it is a special signal handling logic, we may add more document to explain the scenario.

Reply:
Updatd.